sourcegraph/sourcegraph
# SCIM: only support SCIM endpoints on enterprise & business licenses

This adds a new license feature for SCIM and check to ensure the feature is enabled in the SCIM handler.
## Test plan
sg start enterprise 
- verify that SCIM endpoint is accessible 

remove SCIM feature for all plans
- verify that SCIM endpoints all return a forbidden error
